sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationSettingPage.java

```diff
@@ -20,6 +20,13 @@
     @JsonProperty("items")
     private List<ConfigurationSetting> items;
 
+    /**
+     * Creates an instance of {@link ConfigurationSettingPage}
+     */
+    public ConfigurationSettingPage() {
+        // public constructor for graalvm
+    }
```

Comment:
Why is this necessary? There should be a default public ctor already, shouldn't there?

Reply:
Yes, this now works without the constructor. It was required for one of the older dev versions that I was testing with. Removed the ctor now.